nixos/modules/services/networking/teleport.nix

```diff
@@ -0,0 +1,88 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let
+  cfg = config.services.teleport;
+  pidFilePath = "/run/teleport.pid";
+  configJSON = pkgs.writeText "teleport.json" (builtins.toJSON cfg.settings);
```

Comment:
`pkgs.formats.json` is probably better suited here (also as the type for `settings`)

Reply:
This is exactly what I was looking for! Thanks